README.md

```diff
@@ -77,6 +79,22 @@ The supported hooks are listed below. Refer to the [Git documentation](https://g
 - `post-rewrite`
 - `sendemail-validate`
 
+## GIT Large File System Support
```

Comment:
How about:

```
## Git Large File Storage support

If the user has installed [Git Large File Storage](https://git-lfs.github.com/) (`git-lfs`) by calling 
`git lfs install` globally or locally for a repository only, `git-lfs` installs 4 hooks when initializing (`git init`) or cloning (`git clone`) a repository:

- `post-checkout`
- `post-commit`
- `post-merge`
- `pre-push`

Since `githooks` overwrites the hooks in `.git/hooks`, it will also run all *Git LFS* hooks internally if the `git-lfs` executable is found on the system path. You can enforce having `git-lfs` installed on the system by placing a `./githooks/.lfs-required` file inside the repository, then if `git-lfs` is missing, a warning is shown and the hook will exit with code `1`. For some `post-*` hooks this does not mean that the outcome of the git command can be influenced even tough the exit code is `1`, for example `post-commit` hooks can't fail commits. A clone of a repository containing this file might still work but would issue a warning and exit with code `1`, a push - however - will fail if `git-lfs` is missing.

It is advisable for repositories using *Git LFS* to also have a pre-commit hook (e.g. `examples/lfs/pre-commit`) checked in which enforces a correct installation of *Git LFS*.
```

Reply:
Can you just pull and merge your stuff in :-), your native speaker which is in all ways better :-)

Reply:
Sure, I just want to keep your name on the `git blame` for credits. :)
FYI, I am not a native speaker. :)

Reply:
I do it.